test/lib/ufe/testClipboard.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2024 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import fixturesUtils
+import mayaUtils
+import ufeUtils, usdUtils, mayaUtils
+import mayaUsd
+import mayaUsd_createStageWithNewLayer
+
+from maya import standalone
+
+import ufe
+
+import unittest
+
+
+class ClipboardHandlerTestCase(unittest.TestCase):
```

Comment:
Added some generic tests for copy/paste, cut/paste and copy/paste with variant. Once the clipboard design is finalized we can add more unit tests to cover all the design cases.